### DISCUSSION TITLE
Server update Mistlands release

### DISCUSSION THREAD

**Discussion by Daarrz** [lloesche/valheim-server-docker#530] [Q&A]
Hi,

Do i have to update something to get acces to the new Mistlands content ?

Thx

**Comment by jaytark** [comment]
Did you ever get this working?  I updated the latest lloesche docker image and now it says "Incompatible version" when I try to log into my server.

** I noticed several days have pasted since you posted your question and no one's responded.  I'm afraid this project may be dead?
